pymc3/distributions/continuous.py

```diff
@@ -2586,9 +2587,11 @@ class ChiSquared(Gamma):
     nu: int
         Degrees of freedom (nu > 0).
     """
+    rv_op = chisquare
 
-    def __init__(self, nu, *args, **kwargs):
-        self.nu = nu = at.as_tensor_variable(floatX(nu))
+    @classmethod
+    def dist(cls, nu, *args, **kwargs):
+        nu = at.as_tensor_variable(floatX(nu))
         super().__init__(alpha=nu / 2.0, beta=0.5, *args, **kwargs)
```

Comment:
This is the tricky part. The new chisquare op only expects 1 input (`nu`), but then we can't simply use the gamma logp by inheritance as we were doing in V3. 

I think the easiest solution is to implement `logp` and `logcdf` methods that call those from Gamma with the `_logp` and `_logcdf` methods. You can find an example of this in the Zero inflated distributions in `discrete.py`.

With this, this distribution shouldn't inherit from Gamma anymore